.idea/libraries/Maven__ch_qos_logback_logback_classic_1_2_3.xml

```diff
@@ -1,13 +0,0 @@
-<component name="libraryTable">
```

Comment:
Checking in .idea/ files is fine, but I don't think these three files should be removed.